src/Shouldly.Tests/ShouldBeEquivalentTo/ObjectScenario.cs

```diff
@@ -81,6 +81,51 @@ but was
     but was
 ""Bob""
 
+Additional Info:
+    Some additional context");
+        }
+
+        [Fact]
+        public void ShouldFailWhenTitleDoesNotMatch()
+        {
+            var subject = new FakeObject
+            {
+                Id = 5,
+                Title = "Mr",
+                Name = "Bob",
+            };
+            var expected = new FakeObject
+            {
+                Id = 5,
+                Title = "Sir",
```

Comment:
Could either name this `Field`/`TitleField` or initialize only this value so that it is easy to see by looking at the test code where the field is?